src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareConnectionProxy.java

```diff
@@ -318,32 +191,6 @@ protected void initSettings(ConnectionUrl connectionUrl) throws SQLException {
 
     this.enableFailoverSetting =
         connProps.getBooleanProperty(PropertyKey.enableClusterAwareFailover).getValue();
```

Comment:
PropertyKey.enableClusterAwareFailover needs to be reviewed. It makes no sense now. If failover plugin is loaded then a user wants a failover. No extra settings is necessary.

Reply:
We load the plugin if this setting is enabled.

What do you mean by extra settings?